app/views/ex_links/show.html.haml

```diff
@@ -0,0 +1,31 @@
+%p#notice= notice
+
+%p
+  %b
+  = image_tag @ex_link.image.url(:medium)
+
+%p
+  %b Label:
+  = @ex_link.label
+%p
+  %b Url:
+  %a{:href => @ex_link.url, :title => @ex_link.label} #{@ex_link.url}
```

Comment:
`%... \#{expression}` can be written without interpolation as `%...= expression`<br>Hash attribute should start with one space after the opening brace<br>Hash attribute should end with one space before the closing brace